.github/workflows/test.yml

```diff
@@ -128,6 +128,12 @@ jobs:
             run: |
                 php -f nextcloud/occ config:system:set debug --type bool --value true
                 php -f nextcloud/occ config:system:set app.mail.verify-tls-peer --type bool --value false
+          - name: Enable slow mysql query logs
+            if: ${{ matrix.db == 'mysql' }}
+            run: |
+              echo "SET GLOBAL log_queries_not_using_indexes = 1;" | mysql -h 127.0.0.1 -u root -pmy-secret-pw
```

Comment:
on CI queries aren't slow enough to trigger a *slow query* as per https://mariadb.com/kb/en/slow-query-log-overview/, so I've added this line to show anything that doesn't use an index.

Reply:
Any clue if that shows queries where an index exists but is not used because it matches 90% of the rows anyway and stuff like this?

Reply:
That is indeed the case. So if your tests clear a table with `DELETE FROM xyz` it will show up.